tests/foreman/api/test_hostgroup.py

```diff
@@ -5,26 +5,25 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: API
+:CaseComponent: HostGroup
 
 :TestType: Functional
 
 :CaseImportance: High
 
 :Upstream: No
 """
-from fauxfactory import gen_string
-from nailgun import client, entities, entity_fields
 from random import randint
+from fauxfactory import gen_string
```

Comment:
Please set this one in alphabetic order

Reply:
According to PEP8, import order should be:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

So, `random` should be on top. But you are right, `nailgun` should be above `requests`. This is fixed now, and consistent with order with couple of other files I have checked (like `test_computeresource_libvirt.py`).

Reply:
yeah that's true and overall idea was to set in alphabetic order (as you mentioned above order only)